arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/TraceSection.java

```diff
@@ -190,10 +190,9 @@ private List<TraceFragment> makeStackFragments(final Hub hub, CallFrame currentF
     return stackFragments;
   }
 
-  public void triggerHashInfo(Bytes hash) {
+  public void writeHashInfoResult(Bytes hash) {
     for (TraceFragment fragment : this.fragments()) {
       if (fragment instanceof StackFragment) {
-        ((StackFragment) fragment).hashInfoFlag = true;
```

Comment:
This was a mistake: there are times where we get the HASH without triggering the HASH_INFO lookup. E.g. whenever we have to compute `KECCAK(( ))`.